src/framework/testing/qtestsuite.h

```diff
@@ -91,7 +91,7 @@ class QTestSuite
         }
         int ret = 0;
         for (QObject* test : testList()) {
-            if (argc > 0 && test->objectName() == argv[1]) {
+            if (argc > 0 && std::find(argv + 1, argv + argc, test->objectName()) != argv + argc) {
```

Comment:
allows running engraving_tests with a list of class names (e.g. TestTimesig TestTools)